lib/model/CMetricPopulationModel.cc

```diff
@@ -919,6 +927,10 @@ void CMetricPopulationModel::fill(model_t::EFeature feature,
     std::size_t dimension{model_t::dimension(feature)};
     auto data = find(this->featureData(feature, bucketTime), pid, cid);
     const maths::CModel* model{this->model(feature, cid)};
+    if (model == nullptr) {
+        LOG_TRACE(<< "Model unexpectedly null");
+        return;
+    }
```

Comment:
This isn't sufficient. I don't think this should ever happen, but assuming it does this gets dereferenced at on this [line](https://github.com/elastic/ml-cpp/blob/master/lib/model/CMetricPopulationModel.cc#L593). My inclination would be to have these methods return false if the model is missing and take action in the calling code as well.